.github/workflows/ci.yml

```diff
@@ -172,9 +172,15 @@ jobs:
         run: pytest -v py
 
   test_integration:
-    name: Integration Tests
+    name: Integration Tests ${{ matrix.relay_version_chain }}
```

Comment:
Since `relay_chain` is already parameterized, we don't need another job for this. Simply export the env var, and ensure that `relay_chain` runs once without the additional versions, and once with.

Reply:
I've updated the testsuite, lmk

Reply:
There's still interpolation in the job name.